app/lib/server/startup/settings.js

```diff
@@ -2821,4 +2821,15 @@ settings.addGroup('Rate Limiter', function() {
 	});
 });
 
+settings.addGroup('Troubleshoot', function() {
```

Comment:
do we need a disclaimer saying disabling any of the settings can cause malfunction if not used properly or something like that? 

if this is something we'd like to really support I think we'd need to have a better explanation what would be affected by turning off each of the settings. but for now at least a disclaimer saying to "not touch if you don't what you're doing" is enough